### PR TITLE
Add com.eduke32.EDuke32

### PR DESCRIPTION
**Pull request by FakeShemp**
I broke the [old PR](https://github.com/flathub/flathub/pull/600) so lets try this again :)

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build com.eduke32.EDuke32

File: com.eduke32.EDuke32.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  - --device=dri
+  # For storing game files
+  - --filesystem=xdg-documents/EDuke32:create
+  # Unfortunately doesn't seem to respect XDG
+  - --persist=.config/eduke32
+add-extensions:
+  com.eduke32.EDuke32.Plugin:
+    versions: 1-3.32
+    directory: extensions
+    subdirectories: false
```
Comment (gasinvein) [inline com.eduke32.EDuke32.yaml]: Following previous discussion: https://github.com/flathub/flathub/pull/600#discussion_r311439220
`subdirectories: true` is for multiple extensions under the same point, `subdirectories: false` is for single extension.
For the later, extension point should match exact extension ID, e.g. `com.eduke32.EDuke32.DukeNukem3DShareware` both here and for the extension.

File: com.eduke32.EDuke32.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
+app-id: com.eduke32.EDuke32
+runtime: org.gnome.Platform
```
Comment (gasinvein) [inline com.eduke32.EDuke32.yaml]: Does it require something from Gnome runtime? If not, better use FD.o runtime.

Comment (FakeShemp) [reply com.eduke32.EDuke32.yaml]: Ah, I think it used to when I made the old PR, but they seem to have removed the menu it had before, and Gnome along with it. Thanks for spotting that.

File: com.eduke32.EDuke32.yaml (hunk 1 above)
Comment (FakeShemp) [reply com.eduke32.EDuke32.yaml]: I think I understand how it all works now :) Thanks for helping out.

**Comment by FakeShemp** [comment]
bot, build com.eduke32.EDuke32

**Comment by gasinvein** [comment]
Seems like CI status update is stuck.
bot, build com.eduke32.EDuke32

**Comment by bilelmoussaoui** [comment]
bot, build com.eduke32.EDuke32

File: com.eduke32.EDuke32.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
+app-id: com.eduke32.EDuke32
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
```
Comment (LeandroStanger) [inline com.eduke32.EDuke32.yaml]: Update runtime to 19.08

**Comment by FakeShemp** [comment]
bot, build com.eduke32.EDuke32

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.eduke32.EDuke32 Read https://github.com/flathub/flathub/wiki/App-Maintenance for more on how Flathub works for app builders and accept the GitHub request you'll soon be sent!

**Pull request closed without merging**